README.md

```diff
@@ -136,7 +140,7 @@ Now you are ready to deploy Presto on YARN either manually or via Ambari.
 
 ### Manual Installation via Slider
 
-* Download the slider 0.80.0 installation file from http://slider.incubator.apache.org/index.html to one of your nodes in the cluster
+* Download the slider 0.8* installation file from http://slider.incubator.apache.org/index.html to one of your nodes in the cluster (0.8* is the current stable version, this example will use 0.80.0).
```

Comment:
we only support slider-0.80.0, and the code here was never tested for versions above that. So we should mention that here (it is already in the "Requirements" section)